Homework4/Homework4/Validators.cs

```diff
@@ -0,0 +1,26 @@
+using Homework4.Interfaces;
+
+namespace Homework4;
+
+public class Validator<T>
+{
+    private readonly List<IValidator<T>> _validators;
+
+    public bool Validate(T model)
+    {
+        foreach (var validator in _validators)
+        {
+            if (!validator.Validate(model))
+            {
+                return false;
+            }
+        }
+
+        return true;
+    }
```

Comment:
You can enhance this by returning custom ValidationResult, so you can collect list of exceptions or error messages there

Reply:
Fixed